.github/actions/deploy-media-theme/action.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy Media Theme
```

Comment:
I'm not sure about this and the workflow being here vs LGHA, but it feels wrong to add such project-specific GHA code to the general LGHA.